frontend/src/app/patients/Components/ManageEmails.tsx

```diff
@@ -123,7 +123,8 @@ const ManageEmails: React.FC<Props> = ({
           <div className="display-flex">
             <TextInput
               name={`email-${idx}`}
-              className="flex-fill"
+              idString={`email-${idx}`}
+              className="flex-fill emailFormElement"
```

Comment:
Curious why we need this class? 🤔

Reply:
We’re using the classes here to group all the potential error messages in the case of email / phone so we can grab and loop through them when applicable. Line 296 of the PersonForm.tsx file for reference!

Reply:
Gotcha! Thanks for pointing me to that!